### ISSUE TITLE
进行STEP3.训练过程中，验证时出现了问题  During the training process STEP3, problems occurred during the verification

### ISSUE THREAD

**Issue by TaylorRex** [MrGiovanni/DiffTumor#2]
感谢您做出的杰出贡献！在复现您的工作时，进行STEP3训练时，验证过程中发生错误。
应该是在对 logits 进行 softmax 操作时出现了问题 logits = torch.softmax(logits, 1).cpu().numpy()，似乎是发生了索引越界，在多个服务器上尝试均发生相同问题，寻求问题解决的方法。
Thank you for your outstanding contributions! While reproducing your work, an error occurred during the validation process while performing the STEP3 training.
There should be a problem with logits = torch.softmax(logits, 1).cpu().numpy(). It seems that there is an index out of line. The same problem occurs when trying to solve the problem on multiple servers.
![微信图片_20240324155237](https://github.com/MrGiovanni/DiffTumor/assets/100455315/fece13dc-a6c2-482c-869d-514526e85eef)

**Comment by qic999** [comment]
@TaylorRex Thanks for your attention to our work! We have fixed the bug in STEP 3. Please have a try, and if you have any questions, please feel free to let me know.
